.github/workflows/ci.yml

```diff
@@ -5,6 +5,21 @@ name: CI
 on: [push]
 
 jobs:
+  format-cmake:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Format CMake files
+        id: cmake-format
+        uses: PuneetMatharu/cmake-format-lint-action@v1.0.1
+        with:
+          args: --config-files cmake-format.py
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_user_name: cmake-format-bot
+          commit_message: 'Automated commit of cmake-format changes.'
```

Comment:
This checks out the repo, runs `cmake-format` on all files that match `CMakeLists.txt` or `*.cmake`, and then commits any changes that formatter makes.

The rename of `wpe.pc.cmake` to `wpe.pc.in` was because the GitHub action would try and format it.